lib/pacio_test_kit/searcht_test.rb

```diff
@@ -95,6 +95,12 @@ def perform_search_with_system(params)
     end
 
     def search_and_check_response(params, resource_type = self.resource_type)
+      # set params to be lowercase for fhir_search in cases where camelCase
+      params_formatted_case = {}
+      params.each do |key, value|
+        params_formatted_case[key.downcase] = value
+      end
+      params = params_formatted_case
```

Comment:
params must not be reformatted. Ensure you pass the correct parameter name (code) to the test.